package.json

```diff
@@ -4,7 +4,7 @@
         "@types/jest": "^29.5.12",
         "@types/jest-environment-puppeteer": "^5.0.3",
         "@types/jquery": "^3.5.16",
-        "@types/sortablejs": "^1.15.1",
+        "@types/sortablejs": "^1.3.32",
```

Comment:
Should we document why?

Reply:
I think its fine to have the version explicitly listed here, documentation is annoying because `.json` doesn't allow comments.

We should, however, use the "any patch version" marker (`~`), not the "any minor version" (`^`), if we want to keep the versions synced.

Reply:
I meant at least writing somewhere in this PR why we have to use this older version

Reply:
From the first comment, I thought that this would match the version of sortable that we have checked in in our repository

Reply:
Yes, [our static file](https://github.com/e-valuation/EvaP/blob/main/evap/static/js/Sortable.min.js) is version 1.3.0-rc2. There is no release with patch-version 0 for the types package, but since this is for typechecking anyways, I think patch 32 is fine as well.

Note: I also explicitly accepted a higher minor version in another PR in the past -- I'm fine with both approaches. The matching version here is probably less surprising/confusing. We just have to make sure that dependabot doesn't come along and updates this again. I don't think we have an exclusion for this, yet.